chalicelib/responseobjects/hca_response_v5.py

```diff
@@ -273,6 +273,9 @@ def make_processes(self, entry):
         processes = {}
         for bundle in entry["bundles"]:
             for process in bundle["contents"]["processes"]:
+                # HACK: need to check on the indexer why raw protocols are being added
```

Comment:
How will you remember to do this?

Anyhow, this is exactly why we need to merge the two repos. My eyes are burning looking at all this tight coupling between separate projects.